prime-angular-frontend/openshift.nginx.conf

```diff
@@ -32,6 +32,19 @@ server {
     proxy_pass http://$SVC_NAME-webapi:8080/api/PLRHL7;
     #        proxy_set_header  X-SSL-CERT $ssl_client_escaped_cert;
   }
+  location /nginx_status {
```

Comment:
Am I right that this will only be accessible on it's on pod? Shouldn't the cluster have access to this for a status check?